sanic/server.py

```diff
@@ -154,6 +157,8 @@ def data_received(self, data):
             self.headers = []
             self.parser = HttpRequestParser(self)
 
+        self.state['requests_count'] = self.state['requests_count'] + 1
```

Comment:
I tried this line in different places, connection_made, data_received, on_message_complete, etc. 
when i put it in data_received and on_message_complete, looks good to me. However, the request count becomes weird when puts in connection_made.

seems like aiohttp puts it in a similar place. https://github.com/aio-libs/aiohttp/blob/master/aiohttp/web_protocol.py#L296

Anyway, both data_received and on_message_complete look good to me.